launcher/cli/apps.go

```diff
@@ -246,12 +244,8 @@ func init() {
 					userLog.Error(`*********************************************************************************
 * Mindreader failed to start nodeos process
 * To see nodeos logs...
-* DEBUG=\"github.com/dfuse-io/manageos.*\" dfuseeos start
-*********************************************************************************
-
-Make sure you have a dfuse instrumented 'nodeos' binary, follow instructions
-at https://github.com/dfuse-io/dfuse-eosio/blob/develop/INSTALL.md
-to find how to install it.`)
+* DEBUG="mindreader" dfuseeos start
```

Comment:
The message about dfuse instrumented is not required since we check it on `start` by default, so it will be a valid `nodeos` instrumented binary from now on.